.github/workflows/build_make.yml

```diff
@@ -0,0 +1,210 @@
+name: Build Make
+
+# For any push we try to build the project with make
+# When a PR is open we do more test such as test more make rules
+on: [push,pull_request]
+
+env:
+  OCAML_DEFAULT_VERSION: 4.10.0
+
+jobs:
+  make_all:
+    # Basic test to check if the project build with make.
+    # This test also check that .opam files are synced with dune-project
+    # If this test fails, no more make building test are done
+    name: Make all
+
+    strategy:
+      matrix:
+        cache-name:
+          - make-all
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Retrieve opam cache
+        uses: actions/cache@v2
+        id: cache-opam
+        with:
+          path: ~/.opam
+          key: v1-${{ runner.os }}-alt-ergo-${{ matrix.cache-name }}-${{ env.OCAML_DEFAULT_VERSION }}-${{ hashFiles('*.opam') }}
```

Comment:
is it really useful to have the matrix.cache-name part in a variable rather than manually inlined here ? I'd think the only interesting use of a variable is if we one day introduce more different builds in the matrix which need different caches, or do you have another use in mind ?

Reply:
I've used this for previous tests. This is not usefull anymore.